src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -236,6 +239,22 @@ const CreateImageWizard = () => {
           >
             <OscapStep />
           </WizardStep>
+          <WizardStep
+            name="File system configuration"
+            id="step-file-system"
+            footer={
+              <CustomWizardFooter
+                disableNext={
+                  (isFileSystemConfigValid(fsc)?.duplicates?.length !== 0 &&
+                    isFileSystemConfigValid(fsc)?.duplicates?.length !==
+                      undefined) ||
```

Comment:
Could the not zero nor undefined be replaced by `isFileSystemConfigValid(fsc)?.duplicates?.length > 0`

It checks if there are any duplicates, right?